### PR TITLE
Add contribution guidelines

### LINKED ISSUE #3: Add contribution guidelines
Same as title

### PR DESCRIPTION
**Pull request by abhinayagarwal**
Fixes #3

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+## Advice for new contributors
+
+Familiarize yourself with the project's purpose, goals, and architecture.
+Explore the existing documentation, including the README file.
+Check the issue tracker and pull requests to see if your desired contribution is already being discussed or worked on.
+Join our community discussion channels to engage with other contributors and seek guidance if needed.
+
+## Code of Conduct
+
+Please adhere to our project's Code of Conduct when participating in any project-related activities, both online and offline.
```
Comment (jperedadnr) [inline CONTRIBUTING.md]: Where us this code of conduct? Do we have one?? I'd remove this section.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+Treat all community members with respect and create a welcoming and inclusive environment.
+
+## Reporting Issues
+
+If you encounter a bug or have a suggestion, please submit an issue in the project's issue tracker.
```
Comment (jperedadnr) [inline CONTRIBUTING.md:16]: link?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+## Making Contributions
+
+Fork the repository and create a new branch for your contribution.
+Follow the project's coding standards and guidelines.
+Write clear and concise commit messages.
+Test your changes thoroughly.
+Update the documentation if necessary.
+Make sure your code passes all existing tests and linters.
+Submit your contribution as a pull request and link it to an issue with "Fixes #issue-number in the description of the PR.
```
Comment (jperedadnr) [inline CONTRIBUTING.md]: missing `"`

File: README.md [hunk 4]
```diff
@@ -43,10 +43,14 @@ mvn javafx:run
...
 
+There are two common ways to contribute:
 
+- Submit [GitHub issues](https://github.com/gluonhq/emoji/issues) for bug reports, questions, or requests for enhancements.
+- Open a [GitHub pull request](https://github.com/gluonhq/emoji/pulls).
 
-
-
-
+Follow [contributing rules](https://github.com/gluonhq/emoji/blob/master/CONTRIBUTING.md) for this repository.
```
Comment (jperedadnr) [inline README.md:56]: Do we need a CLA?

File: CONTRIBUTING.md (hunk 2 above)
Comment (abhinayagarwal) [reply CONTRIBUTING.md:16]: I haven't added any project specific links here to keep it common for all the repositories. The link for issue tracker and PRs is available in ReadME

**Review by jperedadnr — APPROVED** [review]

**Pull request merged**